components/tidb_query_datatype/src/codec/data_type/vector.rs

```diff
@@ -537,23 +537,23 @@ mod tests {
         assert_eq!(column.capacity(), 3);
         assert!(column.is_empty());
         assert_eq!(column.as_real_slice(), &[]);
-        assert_eq!(column.clone().capacity(), 0);
```

Comment:
According to CI, the unit test is broken after these changes

Reply:
Fixed